### DISCUSSION TITLE
Lazygit closes when using <C-j> or <C-k> to reorder commits

### DISCUSSION THREAD

**Discussion by stefanello57** [LunarVim/LunarVim#3873] [Q&A] [answered]
I'm using Lazygit from within LunarVim `<leader>gg`. When I try to reorder my commits with <C-j> and <C-k> Lazygit is closing instead of reordering the commits. I haven't configured any custom mappings nor is there any default mapping that could cause this.

Any ideas what could be the reason for this behavior?

**Comment by LostInTheLogs** [comment, marked as answer]
it's the default lvim mapping for moving between windows

**Reply by stefanello57** [reply]
How can I disable those mappings? 

Setting them to `false` as described in the docs does not fix the issue:
` lvim.keys.normal_mode["<C-j>"] = false`
` lvim.keys.normal_mode["<C-k>"] = false`

**Reply by LostInTheLogs** [reply]
It's term_mode

**Reply by stefanello57** [reply]
Works! Thanks alot
